spec/system/errors_spec.rb

```diff
@@ -2,27 +2,6 @@
 
 RSpec.describe 'Error pages' do
   context 'when authenticated' do
-    context 'when an unhandled error occurs' do
```

Comment:
Why has this spec been removed?

Reply:
Testing this in the system spec is no longer possible with the show_exception configuration as it is.